modules/programs/quickshell.nix

```diff
@@ -0,0 +1,90 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.programs.quickshell;
+in
+{
+  meta.maintainers = [ lib.hm.maintainers.justdeeevin ];
+
+  options.programs.quickshell = {
+    enable = lib.mkEnableOption "quickshell, a flexbile QtQuick-based desktop shell toolkit.";
+    package = lib.mkPackageOption pkgs "quickshell" { nullable = true; };
+    configs = lib.mkOption {
+      type = lib.types.attrsOf lib.types.path;
+      default = { };
+      description = ''
+        A set of configs to include in the quickshell config directory. The key is the name of the config.
+
+        The config quickshell should use can be specified with the `config` option.
+      '';
+    };
+    activeConfig = lib.mkOption {
```

Comment:
What is the behavior if this is unset? Does it just randomly choose a config? Alphabetical?

Reply:
quickshell will look for `~/.config/quickshell/shell.nix` if no config name is specified with `--config`